dashboard/src/components/Config/AppRepoList/AppRepoAddDockerCreds.tsx

```diff
@@ -67,123 +74,98 @@ export function AppRepoAddDockerCreds({
   };
 
   return (
-    <div className="clr-form-columns">
-      {currentImagePullSecrets.length > 0 ? (
-        currentImagePullSecrets.map(secret => {
+    <>
+      <CdsCheckboxGroup>
+        <label>Associate Docker Registry Credentials (optional)</label>
+        {currentImagePullSecrets.length ? (
+          <CdsControlMessage>
+            Select existing secret(s) to access a private Docker registry and pull images from it.
+            More info{" "}
```

Comment:
moved this text from the AppRepoForm here since the text is actually related to this creds